client/src/main/java/io/pravega/client/netty/impl/ConnectionFactoryImpl.java

```diff
@@ -66,7 +66,7 @@ public ConnectionFactoryImpl(boolean ssl) {
         this.ssl = ssl;
         try {
             this.group = new EpollEventLoopGroup();
-        } catch (ExceptionInInitializerError | NoClassDefFoundError e) {
+        } catch (ExceptionInInitializerError | UnsatisfiedLinkError | NoClassDefFoundError e) {
```

Comment:
This change doesn't belong here, let's do it elsewhere, please.

Reply:
Yes, netty changed the exception it throws.